.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: sh
+install:
+- sudo apt-get update && sudo apt-get install --yes docker
+script:
+- source ./common_functions.sh && for ver in ${supported_versions}; do ./build_latest.sh ${ver}; done
```

Comment:
You can now call the ```./build_all.sh``` script instead of the above commandline as that is exactly what it does. Hopefully that should also take care of the error that you are seeing now.